src/stories/TimePicker24Hour.stories.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { IntlProvider } from "react-intl";
+import moment from "moment";
+import TimePicker24Hour from "../components/TimePicker24Hour/TimePicker24Hour.jsx";
+
+export default {
+  title: "Time Picker 24 hour",
+};
+
+export const Primary = () => {
+  return (
+    <IntlProvider locale={"en"}>
```

Comment:
See if you can make this <IntlProvider> wrapper common for all stories. You should be able to do it through the storybook mechanisms in preview.js I believe or else main.js

Reply:
Done 👍